master/security/kubernetes-default-deny.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Enable default deny for Kubernetes pods
+---
+
+### Big picture
+
+Enable a default deny policy for Kubernetes pods using Kubernetes or Calico network policy.  
+
+### Value
+
+A **default deny** network policy provides an enhanced security posture -- so pods without policy (or incorrect policy) are not allowed traffic until appropriate network policy is defined.
+
+### Features
+
+This how-to guide uses the following Calico features:
+- **NetworkPolicy** 
+- **GlobalNetworkPolicy**
+
+### Concepts
+
+#### Default deny/allow behavior
+
+**Default allow** means all traffic is allowed by default, unless otherwise specified. **Default deny** means all traffic is denied by default, unless explicitly allowed. **Kubernetes pods are default allow**, unless network policy is defined to specify otherwise.
+
+For compatibility with Kubernetes, **Calico network policy** enforcement follows the standard convention for Kubernetes pods:
+- If no network policies apply to a pod, then all traffic to/from that pod is allowed.
+- If one or more network policies apply to a pod with type ingress, then only the ingress traffic specifically allowed by those policies is allowed.
+- If one or more network policies apply to a pod with type egress, then only the egress traffic specifically allowed by those policies is allowed.
+
+For other endpoint types (VMs, host interfaces) that are supported only in **Calico global network policy**, the default behavior is to deny traffic. Only traffic specifically allowed by network policy is allowed, even if no network policies apply to the endpoint.
```

Comment:
> For other endpoint types (VMs, host interfaces) that are supported only in **Calico global network policy**

This isn't right - namespaced network policies can apply to OpenStack VMs as well. GNP is required for host endpoints, though. 

I can address in a follow up PR.